__tests__/fixtures/action/all_fields_readme.input

```diff
@@ -1,14 +1,10 @@
-<!-- action-docs-header action="__tests__/fixtures/all_fields_action.yml" -->
```

Comment:
Is this a mistake to remove the header here?

Reply:
There was a mistake in [this PR](https://github.com/npalm/action-docs/pull/526/files#diff-4fc3464ad7f047a4e7ebab0be7ed9690dda3da44453faeef80ed0af94fdb0351) - this file shouldn't be populated with documentation already so I think this confused me. I'll correct this